### PR TITLE
Limit props to the interface when calling useTabPanel

### PR DESCRIPTION
**Pull request by MarcelReis**
Closes https://github.com/adobe/react-spectrum/issues/1886

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [x] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [x] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [x] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

Open the Storybook in the `Tabs.stories.tsx`, verify the stories 'tabs at the bottom'. The `div` should not contain the HTML attribute `height` with the value "size-1000",

## 🧢 Your Project:

Outside adobe.

### CODE REVIEW

File: packages/@react-aria/tabs/src/useTabPanel.ts
Comment (snowystinger) [inline packages/@react-aria/tabs/src/useTabPanel.ts:66]: so the returned object of useLabels is 
```
  return {
    id,
    'aria-label': label,
    'aria-labelledby': labelledBy
  };
```
so we no longer need to include those three in the second object to mergeProps

```
    tabPanelProps: mergeProps(tabPanelProps, {
      tabIndex,
      role: 'tabpanel',
      'aria-describedby': props['aria-describedby'],
      'aria-details': props['aria-details']
    });
```

Comment (MarcelReis) [reply packages/@react-aria/tabs/src/useTabPanel.ts]: Fixed.

I need to overwrite the `aria-labelledby`, but I could do it in the `useLabels`.

**Review by snowystinger — APPROVED** [review]
thank you!
will get a second review from the team

**Review by devongovett — APPROVED** [review]

**Pull request merged**
